docs-chef-io/content/server/upgrades.md

```diff
@@ -211,6 +211,8 @@ Check the [post upgrade steps](#post-upgrade-steps) if you are upgrading from a
 
 ### External PostgreSQL
 
+External PostgreSQL instructions are provided as a courtesy only.
```

Comment:
I don't think we need this.
```suggestion
```

Reply:
See the [PostgreSQL documentation](https://www.postgresql.org/docs/) for more information.

Reply:
Yup.  I had previously thought that external postgresql people were 'on their own' so I didn't want us to assume any liability.  Ran it by someone else and that wasn't their impression.